deepchecks/nlp/text_data.py

```diff
@@ -588,13 +595,26 @@ def validate_textdata_compatibility(self, other_text_data: 'TextData') -> bool:
 
         return True
 
-    def head(self, n_samples: int = 5) -> pd.DataFrame:
-        """Return a copy of the dataset as a pandas Dataframe with the first n_samples samples."""
+    def head(self, n_samples: int = 5, model_classes: list = None) -> pd.DataFrame:
+        """Return a copy of the dataset as a pandas Dataframe with the first n_samples samples.
+
+        Parameters
+        ----------
+        n_samples : int, default 5
+            Number of samples to return.
+        model_classes : list, default None
+            List of classes names to use for multi-label display. Only used if the dataset is multi-label.
```

Comment:
if is classification but 0,..,n-1?

Reply:
What? you mean if there are no names? then... you don't have to use it.

Reply:
mm right sry